### DISCUSSION TITLE
History stats Hass

### DISCUSSION THREAD

**Discussion by damanti-me** [custom-components/pyscript#219] [Q&A] [answered]
Hey there,
is there a way to access the history values from HA? Its working with the REstAPI. 
Thanks for your time, i couldnt find the entry...

**Comment by craigbarratt** [comment, marked as answer]
There isn't any direct support, but perhaps you can access what you need via the `hass` variable>. It is available as a global variable if you set the `hass_is_global` config variable.

**Reply by damanti-me** [reply]
Hi, thanks for your reply. I managed it by directly access via RESTApi within the python script. Not beautiful I guess but it works. I'll give your solution a try. 
👍🏼
